pkg/api/norman/store/cluster/cluster_store.go

```diff
@@ -287,6 +287,8 @@ func loadDataFromTemplate(clusterTemplateRevision *v3.ClusterTemplateRevision, c
 
 	defaultedAnswers := make(map[string]string)
 
+	registryConfigPath := fmt.Sprintf("%s.%s", managementv3.ClusterSpecFieldRancherKubernetesEngineConfig, managementv3.RancherKubernetesEngineConfigFieldPrivateRegistries)
```

Comment:
I would use strings.Join instead.  `fmt.Sprintf` relies on reflection making it more expensive.

Reply:
Good point! 
fixd.